scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionBinomial.scala

```diff
@@ -0,0 +1,123 @@
+            pred.classProbabilities(0),
+            pred.classProbabilities(1),
+            pred.calibratedClassProbabilities(0),
+            pred.calibratedClassProbabilities(1),
+            pred.contributions
+          )
+        }
+      } else {
+        udf[WithCalibration, Row] { r: Row =>
+          val pred = predictWrapper.predictBinomial(RowConverter.toH2ORowData(r))
+          WithCalibration(
+            pred.classProbabilities(0),
+            pred.classProbabilities(1),
+            pred.calibratedClassProbabilities(0),
+            pred.calibratedClassProbabilities(1)
+          )
+        }
+      }
+    } else if (model.getWithDetailedPredictionCol()) {
+      udf[WithContribution, Row] { r: Row =>
+        val pred = predictWrapper.predictBinomial(RowConverter.toH2ORowData(r))
+        WithContribution(
+          pred.classProbabilities(0),
+          pred.classProbabilities(1),
+          pred.contributions
+        )
+      }
+    } else {
+      udf[Base, Row] { r: Row =>
+        val pred = predictWrapper.predictBinomial(RowConverter.toH2ORowData(r))
+        Base(
+          pred.classProbabilities(0),
+          pred.classProbabilities(1)
+        )
+      }
+    }
+  }
+
+  private val binomialSchemaBase = Seq("p0", "p1").map(StructField(_, DoubleType, nullable = false))
+
+  override def getPredictionColSchema(@transient model: H2OMOJOModel, @transient predictWrapper: EasyPredictModelWrapper): Seq[StructField] = {
+    if (supportsCalibratedProbabilities(predictWrapper)) {
+      binomialSchemaBase ++ Seq("p0_calibrated", "p1_calibrated").map(StructField(_, DoubleType, nullable = false))
+    } else {
+      binomialSchemaBase
+    }
+  }
+
+  override def getDetailedPredictionColSchema(@transient model: H2OMOJOModel, @transient predictWrapper: EasyPredictModelWrapper): Seq[StructField] = {
+    if (supportsCalibratedProbabilities(predictWrapper)) {
+      val base = binomialSchemaBase ++ Seq("p0_calibrated", "p1_calibrated").map(StructField(_, DoubleType, nullable = false))
+      if (model.getWithDetailedPredictionCol()) {
+        base ++ Seq(StructField("contributions", ArrayType(FloatType)))
+      } else {
+        base
+      }
+    } else if (model.getWithDetailedPredictionCol()) {
+      binomialSchemaBase ++ Seq(StructField("contributions", ArrayType(FloatType)))
+    } else {
+      binomialSchemaBase
+    }
+  }
+
+  case class Base(p0: Double, p1: Double)
+
+  case class WithCalibration(p0: Double, p1: Double, p0_calibrated: Double, p1_calibrated: Double)
+
+  case class WithContribution(p0: Double, p1: Double, contributions: Array[Float])
+
+  case class WithCalibrationAndContribution(p0: Double, p1: Double, p0_calibrated: Double, p1_calibrated: Double, contributions: Array[Float])
+
+  override def extractSimplePredictionColumns(@transient model: H2OMOJOModel, @transient predictWrapper: EasyPredictModelWrapper): Column = {
+    val p0col = col(s"${model.getDetailedPredictionCol()}.p0").as("p0")
```

Comment:
I wanted to put the expose the new details ( such as contributions) to detailed_column so this is exposed in the right place already, instead of exposing to prediction column and moving later. We still need to move for example calibrated probabilities to detailed_prediction column